docs/azure/services/Microsoft_Purview_Information_Protection_Overview.md

```diff
@@ -0,0 +1,96 @@
+# Microsoft Purview Information Protection
+
+Microsoft Purview Information Protection is Microsoft's offering used to discover, classify, protect, and govern sensitive information both in transit and at rest (including the data stored on-premise and in cloud). The Information Protection capabilities are part of the wider Purview risk & compliance offering, accessible in the [Microsoft Purview compliance portal](https://compliance.microsoft.com/homepage).
+
+Microsoft Purview Information Protection is comprised of the following standalone products and uses their capabilities as part of the offering:
+- Azure Information Protection
+- Microsoft 365 Information Protection, such as Microsoft 365 DLP
+- Windows Information Protection
+- Microsoft Defender for Cloud Apps
+
+Enterprises dealing with sensitive information such as financial data, healthcare-related and other personal records are required to adequately record and govern how this data is handled. As part of this process, the system should classify, label and control access to data in accordance with regulatory requirements. 
+
+Due to the nature of large organisations, various bits of sensitive data can end up scattered across on-premise servers, employee devices, cloud-native applications, third-party applications and many other resources. An illustrative diagram from Microsoft's [documentation](https://learn.microsoft.com/en-us/microsoft-365/compliance/information-protection?view=o365-worldwide) shows the high-level capabilities offered by Microsoft Purview Information Protection.
+
+![](../images/PurviewOverview.png)
```

Comment:
a general comment on the images is that they seem fairly small low-res versus the ones that are in the official documentation. Can we make sure that we get a better image if we're using and to ensure we also reference the page from which it comes from. I've tried to do that for the existing images but double check them.